SSSwiftUIAnimations/Examples/ExampleReactionAnimationView.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  ExampleReactionAnimationView.swift
+//  SSSwiftUIAnimations
+//
+//  Created by Faaiz Daglawala on 07/11/24.
+//
+
+import SwiftUI
+
+struct ExampleReactionAnimationView: View {
+    
+    // MARK: - Variables
+    
+    var body: some View {
+        VStack {
+            SSReactionAnimationView(
+                viewModel: SSReactionAnimationViewModel(
```

Comment:
Can we make a StyleClass for providing the Styling Attributes? We can keep the Viewmodel for Logical part of animation.